Content - Mentor Offers their Apprentice Wisdom

Artyphex opened the pull request:
An apprentice has a moment of severe self-doubt are offered "wisdom" by their mentor based on that mentor's personality. All personality types divided up between six different success options

Artyphex committed:
Content - Mentor Offers their Apprentice Wisdom

An apprentice has a moment of severe self-doubt are are offered "wisdom" by their mentor based on that mentor's personality. All personality types dived up between six different success options

Artyphex committed:
Content - Mentor Offers their Apprentice Wisdom

Fixed incorrect capitalization in one of the pronoun tags

Tiritirimatangi commented on resources/dicts/patrols/plains/training/any.json:
```diff
@@ -2828,5 +2828,125 @@
+        "min_cats": 2,
+        "max_cats": 2,
+        "min_max_status": {
+            "normal adult": [0, 1],
```
min max statuses need to be [1, 1]

Tiritirimatangi commented on resources/dicts/patrols/plains/training/any.json:
```diff
@@ -2828,5 +2828,125 @@
+    
+        "weight": 20,
+        "chance_of_success": 50,
+        "intro_text": "p_l is surprised when app1 makes a mistake on a simple move p_l thought {PRONOUN/app1/subject} had mastered. Before p_l can say anything, app1 wails and runs out of the training grounds. When p_l finds {PRONOUN/app1/poss} huddled in camp, {PRONOUN/app1/subject} {VERB/app1/sob/sobs} out a confession: {PRONOUN/app1/subject} will never be ready to be a warrior.",
```
You may want to avoid the word 'wail', as it makes the apprentice sound very young, not what I'd expect from a 12 moon old cat - tho very appropriate for a 6 moon old one. But the wording has to work for both

Tiritirimatangi commented on resources/dicts/patrols/plains/training/any.json:
```diff
@@ -2828,5 +2828,125 @@
+        "weight": 20,
+        "chance_of_success": 50,
+        "intro_text": "p_l is surprised when app1 makes a mistake on a simple move p_l thought {PRONOUN/app1/subject} had mastered. Before p_l can say anything, app1 wails and runs out of the training grounds. When p_l finds {PRONOUN/app1/poss} huddled in camp, {PRONOUN/app1/subject} {VERB/app1/sob/sobs} out a confession: {PRONOUN/app1/subject} will never be ready to be a warrior.",
+        "decline_text": "It wouldn't help to talk to app1 while {PRONOUN/app1/subject} {VERB/app1/is/are} in this state. p_l tells {PRONOUN/app1/poss} {PRONOUN/p_l/subject}'ll leave {PRONOUN/app1/poss} alone, and to come find {PRONOUN/p_l/poss} if {PRONOUN/app1/subject}'d like to keep training today. app1 never does.",
```
{VERB/app1/is/are} --> {VERB/app1/are/is} (the plural one comes first)

tells {PRONOUN/app1/poss} = tells their. Instead, tells {PRONOUN/app1/object} = tells them
https://docs.google.com/document/d/1qcks1E25NkOfU6zdwUlccU42VRkHv_WNaCsMpUP5jDA/edit
for pronoun help 
the poss vs object error continues in this text, please fix

I really like this decline text, I think you did a great job with it and it's a lot better than most of the ones I make lol.

Tiritirimatangi commented on resources/dicts/patrols/plains/training/any.json:
```diff
@@ -2828,5 +2828,125 @@
+            {
+                "text": "p_l waits patiently for app1 to calm down. Once {PRONOUN/app1/subject} can listen, p_l tells {PRONOUN/app1/poss} a secret: no warrior has ever felt like a warrior. Not even p_l. You will always feel like you, p_l says, but don't be afraid. You <i>must</i> be you before you are a warrior. That is what c_n truly needs. You. ",
+                "exp": 10,
+                "stat_trait": ["loving","compassionate","wise","calm","thoughtful","faithful","loyal","sincere","righteous"],
```
I adore this text but you're missing a field here, the "can_have_stat": ["p_l"], which will tell the code that only p_l can trigger this stat outcome, not app1. You also need to refer to p_l as s_c here rather than p_l. This is because it makes finding typos in teh code easier and keeps things consistent :)

Tiritirimatangi commented on resources/dicts/patrols/plains/training/any.json:
```diff
@@ -2828,5 +2828,125 @@
+                "weight": 20
+            },
+            {
+                "text": "App1 sniffles alone for a few moments before facing {PRONOUN/app1/poss} mentor. App1's eyes widen at p_l standing over {PRONOUN/app1/poss}, claws unsheathed and fur bristled. <i>That</i> is weakness. c_n, and more importantly, p_l will not tolerate weakness. Get back into training, app1, and do not let p_l see you act in this way again.",
```
app1 can't be capitalised (it won't be replaced in the text properly)

also needs the "can_have_stat": ["p_l"], and s_c reference in text rather than p_l.

I think that the claws unsheathed wording implies a little too much violence. Can we use out body language instead? Like maybe ears pinned back and fur bristled? Or tail lashing?

Tiritirimatangi commented on resources/dicts/patrols/plains/training/any.json:
```diff
@@ -2828,5 +2828,125 @@
+                "weight": 20
+            },
+            {
+                "text": "p_l's whiskers twitch in annoyance at this childish outburst. If app1 doesn't <i>want</i> to be a warrior, that's fine by p_l. {PRONOUN/p_l/subject}'ll be in the training grounds, waiting to train the greatest warrior c_n's ever seen. {PRONOUN/p_l/subject} {VERB/p_l/walk/walks} away from app1, and a hidden smile dashes across {PRONOUN/p_l/poss} muzzle as {PRONOUN/p_l/subj ...
```
also needs the "can_have_stat": ["p_l"], and s_c reference in text rather than p_l

I love this angle for these personalities!

Tiritirimatangi commented on resources/dicts/patrols/plains/training/any.json:
```diff
@@ -2828,5 +2828,125 @@
+                "weight": 20
+            },
+            {
+                "text": "p_l lashes {PRONOUN/p_l/poss} tail. Of course app1 will never be a warrior if {PRONOUN/app1/subject} {VERB/app1/think/thinks} like <i>that.</i> c_n doesn't want warriors who think like <i>that.</i> Half of glory, {PRONOUN/p_l/subject} {VERB/p_l/explain/explains} to app1, is believing you deserve it. P_l challenges app1: focus on <i>that</i>. When p_l and app1 return to th ...
```
also needs the "can_have_stat": ["p_l"], and s_c reference in text rather than p_l

p_l can't be capitalised - it won't replace with the warrior name properly, it's case sensitive

this is a great outcome and I really like it!

Tiritirimatangi commented on resources/dicts/patrols/plains/training/any.json:
```diff
@@ -2828,5 +2828,125 @@
+                "weight": 20
+            },
+            {
+                "text": "p_l's heart lurches seeing the apprentice so low. Could p_l have done something to make app1 feel this way? Not having any words p_l sits close to app1, wrapping {PRONOUN/p_l/poss} tail around {PRONOUN/app1/poss}. P_l purrs softly, reassuringly. Until app1 purrs back, gratefully. ",
```
awww so cute! 
also needs the "can_have_stat": ["p_l"], and s_c reference in text rather than p_l
can't capitalise p_l

Tiritirimatangi commented on resources/dicts/patrols/plains/training/any.json:
```diff
@@ -2828,5 +2828,125 @@
+                "weight": 20
+            },
+            {
+                "text": "p_l shifts awkwardly at app1's side until {PRONOUN/p_l/subject} can't stand the silence any longer. P_l drops into a crouch and nudges app1 with {PRONOUN/p_l/poss} paw. App1 looks back, confused as p_l keeps bouncing and poking at {PRONOUN/p_l/poss} with a goofy expression on {PRONOUN/p_l/poss} face, until app1 gives in. The warrior and apprentice play like kits, and the  ...
```
SO CUTE

might need to be a little shorter tho

also needs the "can_have_stat": ["p_l"], and s_c reference in text rather than p_l

can't capitalise app1 or p_l

Tiritirimatangi commented on resources/dicts/patrols/plains/training/any.json:
```diff
@@ -2828,5 +2828,125 @@
+        ],
+        "fail_outcomes": [
+            {
+                "text": "P_l tries to think of something to say or do, anything, but {PRONOUN/p_l/subject} fumble as app1 becomes more and more unconsolable. Finally, p_l gives up. Leaving app1 alone with {PRONOUN/app1/poss} thoughts.",
```
lol p_l you aint't a great mentor huh

also needs the "can_have_stat": ["p_l"], and s_c reference in text rather than p_l
can't capitalise p_l

Tiritirimatangi reviewed (changes requested):
I've commented on the specific changes it needs. Overall, I really like the direction your went with this, I think heaps of people are going to love this!

Artyphex committed:
Content - Mentor's Wisdom - Update 1

Fixed pronoun tags, min max cat status, some wording, and added the can_have_stat tag

Tiritirimatangi commented on resources/dicts/patrols/plains/training/any.json:
```diff
@@ -2828,5 +2828,132 @@
+        "weight": 20,
+        "chance_of_success": 50,
+        "intro_text": "p_l is surprised when app1 makes a mistake on a simple move p_l thought {PRONOUN/app1/subject} had mastered. Before p_l can say anything, app1 runs out of the training grounds. When p_l finds {PRONOUN/app1/object} huddled in camp, {PRONOUN/app1/subject} {VERB/app1/sobs/sob} out a confession: {PRONOUN/app1/subject} will never be ready to be a warrior.",
+        "decline_text": "It wouldn't help to talk to app1 while {PRONOUN/app1/subject} {VERB/app1/are/is} in this state. p_l tells {PRONOUN/app1/poss} {PRONOUN/p_l/subject}'ll leave {PRONOUN/app1/object} alone, and to come find {PRONOUN/p_l/object} if {PRONOUN/app1/subject}'d like to keep training today. app1 never does.",
```
p_l tells {PRONOUN/app1/poss} --> p_l tells {PRONOUN/app1/object}

Tiritirimatangi commented on resources/dicts/patrols/plains/training/any.json:
```diff
@@ -2828,5 +2828,132 @@
+        "decline_text": "It wouldn't help to talk to app1 while {PRONOUN/app1/subject} {VERB/app1/are/is} in this state. p_l tells {PRONOUN/app1/poss} {PRONOUN/p_l/subject}'ll leave {PRONOUN/app1/object} alone, and to come find {PRONOUN/p_l/object} if {PRONOUN/app1/subject}'d like to keep training today. app1 never does.",
+        "success_outcomes": [
+            {
+                "text": "s_c waits patiently for app1 to calm down. Once {PRONOUN/app1/subject} can listen, s_c tells {PRONOUN/app1/poss} a secret: no warrior has ever felt like a warrior. Not even s_c. You will always feel like you, s_c says, but don't be afraid. You <i>must</i> be you before you are a warrior. That is what c_n truly needs. You. ",
```
s_c tells {PRONOUN/app1/poss} --> s_c tells {PRONOUN/app1/object}

Tiritirimatangi commented on resources/dicts/patrols/plains/training/any.json:
```diff
@@ -2828,5 +2828,132 @@
+                "weight": 20
+            },
+            {
+                "text": "s_c shifts awkwardly at app1's side until {PRONOUN/s_c/subject} can't stand the silence any longer. s_c drops into a crouch and nudges app1 with {PRONOUN/s_c/poss} paw. App1 looks back, confused as s_c keeps bouncing and poking at {PRONOUN/s_c/object} with a goofy expression on {PRONOUN/s_c/poss} face, until app1 gives in. The warrior and apprentice play like kits, and th ...
```
{VERB/app1/has/have} error

Either, {PRONOUN/app1/subject} {VERB/app1/have/has} or {PRONOUN/app1/subject}{VERB/app1/'ve/'s} to fix

Tiritirimatangi commented on resources/dicts/patrols/plains/training/any.json:
```diff
@@ -2828,5 +2828,132 @@
+        ],
+        "fail_outcomes": [
+            {
+                "text": "s_c tries to think of something to say or do, anything, but {PRONOUN/s_c/subject} {VERB/s_c/fumbles/fumble} as app1 becomes more and more unconsolable. Finally, s_c gives up. Leaving app1 alone with {PRONOUN/app1/poss} thoughts.",
```
Finally, s_c gives up, leaving app1 alone with {PRONOUN/app1/poss} thoughts. (comma in the middle reads slightly better?)

Tiritirimatangi commented on resources/dicts/patrols/plains/training/any.json:
```diff
@@ -2828,5 +2828,132 @@
+        ]
     }
+    
 ]
```
SQUARE BRACKET ALERT

Tiritirimatangi replied:
2955's } encloses failure outcome number 1. 2956's ] encloses the failure outcome block as a whole. Then 2957's } encloses the patrol code for this patrol entirely

Artyphex committed:
Content - Mentor's Wisdom - Update 2

Fixed some object pronoun tags incorrectly marked as poss, fixed a verb tag to the proper order
